client/src/store/new-request-module.ts

```diff
@@ -816,16 +874,44 @@ export class NewRequestModule extends VuexModule {
   get locationOptions () {
     let options = [
       { text: 'BC', value: 'BC' },
-      { text: 'Canada', value: 'CA' },
-      { text: 'Foreign', value: 'IN' },
-      { text: 'Help', value: 'INFO' }
+      {
+        text: 'Extra provincial (Canada based)',
+        value: 'CA',
+        blurb: [
+          `Your existing business is currently located in any Province or Territory other than BC.`,
+          `Your existing business is currently located in any Province or Territory other than BC.`,
```

Comment:
Duplicate blurb?

Reply:
Yes and no 😛 
Each index is for a specific scenario.. some scenarios share the same description. It was easier to just assign the blurb twice where required opposed to write some conditional logic to to apply the same description twice under xxx circumstances.

Reply:
OK, thanks for explaining. Short comment in there maybe?

Reply:
Will do!